.travis/run_travis_job.sh

```diff
@@ -53,4 +53,25 @@ else
     if [[ "${TRAVIS_JOB_NAME}" == "Linux JDK 8 CHECK_RUST=false" ]]; then
       mvn org.eluder.coveralls:coveralls-maven-plugin:report
     fi
+
+    # Test exonum_launcher_java_plugins
+    if [[ "${TRAVIS_JOB_NAME}" == "Linux JDK 8 CHECK_RUST=false" ]]; then
+      cd "${TRAVIS_BUILD_DIR}/exonum-java-binding/exonum_launcher_java_plugins"
+      # Install pip
+      curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+      python3.7 get-pip.py --user
+      # Install exonum-launcher
+      git clone https://github.com/exonum/exonum-launcher.git
```

Comment:
Shallow clone?

Reply:
Actually installing via requirements.txt works well - now we depend on a particular exonum-launcher revision.